src/commands/instance/kill.ts

```diff
@@ -0,0 +1,83 @@
+import { flags } from '@oclif/command'
+import inquirer from 'inquirer'
+import execa from 'execa'
+
+import { Kommand } from '../../common'
+
+export class InstanceLogs extends Kommand {
+  static initSdk = false
+
+  static description = 'Kill all the containers of a running kuzzle instance'
+
+  static flags = {
+    instance: flags.string({
+      char: 'i',
+      description: 'Kuzzle instance name'
+    })
+  }
+
+  async runSafe() {
+    let instance: string = this.flags.instance!
```

Comment:
What does the `!` do at the end?

Reply:
https://www.logicbig.com/tutorials/misc/typescript/non-null-assertion-operator.html

Reply:
You should remove it, it's useless since you can't ensure that this flag is not null. It's only set (or not) at runtime